mne/time_frequency/psd.py

```diff
@@ -112,6 +114,202 @@ def _check_nfft(n, n_fft, n_overlap):
+                   bandwidth=None, adaptive=False, low_bias=True,
+                   normalization='length', picks=None, proj=False,
+                   n_jobs=1, verbose=None):
+    """Compute the power spectral density (PSD) using multitapers.
+
+    Calculates spectral density for orthogonal tapers, then averages them
+    together for each channel/epoch. See [1] for a description of the tapers
+    and [2] for the general method.
+
+    Parameters
+    ----------
+    inst : instance of Epochs or Raw or Evoked
+        The data for PSD calculation.
+    fmin : float
+        Min frequency of interest
+    fmax : float
+        Max frequency of interest
+    tmin : float | None
+        Min time of interest
+    tmax : float | None
+        Max time of interest
+    bandwidth : float
+        The bandwidth of the multi taper windowing function in Hz.
+    adaptive : bool
+        Use adaptive weights to combine the tapered spectra into PSD
+        (slow, use n_jobs >> 1 to speed up computation).
+    low_bias : bool
+        Only use tapers with more than 90% spectral concentration within
+        bandwidth.
+    normalization : str
+        Either "full" or "length" (default). If "full", the PSD will
+        be normalized by the sampling rate as well as the length of
+        the signal (as in nitime).
+    picks : array-like of int | None
+        The selection of channels to include in the computation.
+        If None, take all channels.
+    proj : bool
+        Apply SSP projection vectors. If inst is ndarray this is not used.
+    n_jobs : int
+        Number of CPUs to use in the computation.
+    verbose : bool, str, int, or None
+        If not None, override default verbose level (see mne.verbose).
+
+    Returns
+    -------
+    psds : ndarray, shape ([n_epochs], n_channels, n_freqs)
+        The power spectral densities. If Raw is provided,
+        then psds will be 2-D.
+    freqs : ndarray, shape (n_freqs)
+        The frequencies.
+
+    References
+    ----------
+    .. [1] Slepian, D. "Prolate spheroidal wave functions, Fourier analysis,
+           and uncertainty V: The discrete case." Bell System Technical
+           Journal, vol. 57, 1978.
+
+    .. [2] Percival D.B. and Walden A.T. "Spectral Analysis for Physical
+           Applications: Multitaper and Conventional Univariate Techniques."
+           Cambridge University Press, 1993.
+
+    See Also
+    --------
+    mne.io.Raw.plot_psd, mne.Epochs.plot_psd, psd_welch
+    """
+    # Prep data
+    data, sfreq = _check_psd_data(inst, tmin, tmax, picks, proj)
+    return _psd_multitaper(data, sfreq, fmin=fmin, fmax=fmax,
+                           bandwidth=bandwidth, adaptive=adaptive,
+                           low_bias=low_bias,
+                           normalization=normalization,  n_jobs=n_jobs,
+                           verbose=verbose)
+
+
+@deprecated('This will be deprecated in release v0.13, see psd_ functions.')
```

Comment:
I would point to `psd_welch` specifically because that does exactly the same as this.